onnx/optimizer/passes/eliminate_unused_initializer.h

```diff
@@ -0,0 +1,73 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
```

Comment:
Nit: add some example to explain the behavior of this pass, something like https://github.com/onnx/onnx/blob/master/onnx/optimizer/passes/fuse_add_bias_into_conv.h#L6